### DISCUSSION TITLE
Any way to remove ellipses only from the end of lines?

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5383] [General]
Trying to remove (...) But only from the end of lines, not from the middle of a sentence.

**Comment by niksedk** [comment]
You could try Edit - Multiple replace: is an regex for find with a value of `...$` and use empty for replace string.

**Reply by DrReddP** [reply]
Could you show me a screenshot of exactly what to do? I don't understand the instructions
